ui/src/app/base/hooks.ts

```diff
@@ -17,6 +17,12 @@ import { actions as messageActions } from "app/store/message";
 import type { RootState } from "app/store/root/types";
 import { kebabToCamelCase } from "app/utils";
 
+declare global {
+  interface Window {
+    ga: (...args: unknown[]) => void;
```

Comment:
Not sure why this randomly started complaining at me (it uses `window.ga` in one of the hooks in this file).